lrathoni/Lullaby-for-a-fish
# Moved large files out of GIT repository.

### What

Basically reorganised large files; moved textures, models, audio in `Assets/Resources` sub-folders and uploaded them to Dropbox, so that we no longer need Git LFS.

These files are available [here](https://www.dropbox.com/sh/6oh6xv1yqpieuvr/AAB28qTgaEP2JC07N8jAbiHTa?dl=0).

* Moved some texture files out of `Assets/Resources/Models` and into the appropriate folder.
* Moved the texture folder `Assets/Textures` to `Assets/Resources/Textures`.
* Moved `Assets/Resources/Prefabs` to `Assets/Prefabs`. Unity Prefabs are text files, and should still be handled by GIT.
* Removed `Assets/Resources` in Unity project, (you can find it in the Dropbox shared folder).

### How to test

1. Download the Dropbox shared folder.
2. Put the folder in `LullabyProject`.
3. Launch Unity and check that there are no missing files or broken links.
